libcaf_test/caf/test/fixture/deterministic.hpp

```diff
@@ -379,6 +446,25 @@ class CAF_TEST_EXPORT deterministic {
     return evaluator<Ts...>{this, loc, evaluator_algorithm::allow};
   }
 
+  /// Helper class for `inject` that only provides `with`.
+  struct inject_helper {
+    deterministic*fix;
```

Comment:
This should fix the `clang-format`
```suggestion
    deterministic* fix;
```